tests/install_vm.py

```diff
@@ -39,7 +39,18 @@ def parse_args():
         "--disk-dir",
         dest="disk_dir",
         default=None,
-        help="Location of the VM qcow2 file."
+        help="Location of the VM qcow2 disk file (ignored when --disk is specified)."
+    )
+    parser.add_argument(
+        "--disk-size",
+        dest="disk_size",
+        default=20,
+        help="Size of the VM qcow2 disk, default is 20 GiB (ignored when --disk is specified)."
```

Comment:
Instead of ignoring you can try marking them as mutually exclusive group: https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.add_mutually_exclusive_group

Reply:
The `--disk-dir` and `--disk-size` options can be used together, but when `--disk` option is specified it will have higher priority and will overwrite the other two. This is not possible to implement using `ArgumentParser.add_mutually_exclusive_group`, as it would need to work with subgroup of arguments (because of `--disk-dir` and `--disk-size` not being mutually exclusive).

Reply:
ok